core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/DetailedSampleBuilder.java

```diff
@@ -60,46 +62,52 @@
   private User lastModifier;
   private Double volume;
   private boolean emptied = false;
-  
+
   // DetailedSample attributes
-  private Sample parent;
+  private SampleAdditionalInfo parent;
   private SampleClass sampleClass;
   private QcPassedDetail qcPassedDetail;
   private Subproject subproject;
   private Long kitDescriptorId;
   private KitDescriptor prepKit;
-  private Double concentration;
   private Boolean archived = Boolean.FALSE;
   private Long groupId;
   private String groupDescription;
-  
+
   // Identity attributes
   private String internalName;
   private String externalName;
   private DonorSex donorSex = DonorSex.UNKNOWN;
-  
+
   // TissueSample attributes
-  private SampleClass tissueClass; // identifies a parent tissue class if this sample itself is not a tissue
+  private SampleClass tissueClass;
   private Integer cellularity;
+
+  // Transitioning to tissue
   private TissueOrigin tissueOrigin;
   private TissueType tissueType;
   private String externalInstituteIdentifier;
   private Lab lab;
   private Integer passageNumber;
   private Integer timesReceived;
   private Integer tubeNumber;
-  
+
   // AnalyteSample attributes
   private SamplePurpose samplePurpose;
   private TissueMaterial tissueMaterial;
+
+  // AnalyteSample attributes
```

Comment:
Comments no longer accurate / need rearranged (these divisions are helpful to see though)